coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017-2023 LinkedIn Corporation. All rights reserved.
```

Comment:
In the javadoc of this class, I still see a remaining `Calcite2TrinoUDFConverter`, would be great to remove that one too:
```
   * (2) Some internally registered UDFs which should not be converted, like `to_date`.
   *     If the value of key {@link CoralTrinoConfigKeys#AVOID_TRANSFORM_TO_DATE_UDF} is set to true, we don't transform `to_date` UDF
   *     in {@link com.linkedin.coral.trino.rel2trino.Calcite2TrinoUDFConverter.TrinoRexConverter#visitCall(RexCall)}
```

Reply:
Good catch, fixed!